src/terminal.zig

```diff
@@ -87,26 +88,21 @@ pub fn cursorPosition(stdout: fs.File, stdin: fs.File) !Pos {
 
 fn enableRawMode(file: fs.File) !void {
     var raw = try getAttr(file);
-    raw.c_iflag &= ~@as(@TypeOf(raw.c_lflag), BRKINT | ICRNL | INPCK | ISTRIP | IXON);
-    raw.c_oflag &= ~@as(@TypeOf(raw.c_lflag), c.OPOST);
-    raw.c_cflag &= ~@as(@TypeOf(raw.c_lflag), c.CS8);
-    raw.c_lflag &= ~@as(@TypeOf(raw.c_lflag), c.ECHO | c.ICANON | c.IEXTEN | c.ISIG);
-    raw.c_cc[c.VMIN] = 0;
-    raw.c_cc[c.VTIME] = 1;
+    raw.iflag &= ~@as(@TypeOf(raw.lflag), BRKINT | ICRNL | INPCK | ISTRIP | IXON);
+    raw.oflag &= ~@as(@TypeOf(raw.lflag), os.OPOST);
+    raw.cflag &= ~@as(@TypeOf(raw.lflag), os.CS8);
+    raw.lflag &= ~@as(@TypeOf(raw.lflag), os.ECHO | os.ICANON | os.IEXTEN | os.ISIG);
+    raw.cc[5] = 0;
+    raw.cc[7] = 1;
```

Comment:
I am not so sure about these since there are no VMIN and VTIME in the Zig standard library.

Reply:
Hmm. No idea. Let's at least define them in this file then, so that we know these are meant to be `VMIN` and `VTIME`.

Reply:
I was looking for more information and found both this pull request: https://github.com/ziglang/zig/pull/5585 and I found the declaration of VTIME and VMIN in the musl source code here: https://git.musl-libc.org/cgit/musl/tree/arch/generic/bits/termios.h

I think it is safe to make variables with `const VTIME = 5`and `const VMIN = 6;`.

Reply:
I see that you already have done that? Well, I used the variables that were already in the file. I guess that research wasn't so necessary.